obspy/signal/spectral_estimation.py

```diff
@@ -1824,6 +1915,31 @@ def plot(self, filename=None, show_coverage=True, show_histogram=True,
                 else:
                     xdata = periods
                 ax.plot(xdata, noise_model, '0.4', linewidth=2, zorder=10)
+        
+        if show_earthquakes:
+            for (magnitude, distance), (frequencies, power) in earthquakes.items():
+                # Eq.1 Cauzzi & Clinton (2013)
+                power = np.array(power)
+                frequencies = np.array(frequencies)
+                periods = 1.0 / frequencies
+                ydata = power / (periods ** (-.5))
+                ydata = 20 * np.log10(ydata / 2)
+                if (eqlims[0] < magnitude and magnitude < eqlims[1] and
+                        eqlims[2] < distance and distance < eqlims[3] and
+                        min(ydata) < self.db_bin_edges[-1]):
```

Comment:
Slightly better readability if you negate the if clause and do a continue, I think. (For a second I was wondering why the xdata is only set in the if clause until I realized the whole plot is in the if clause). That also saves you one indentation level. This is really just cosmetics.. but still, the better the shape of the code, the easier to maintain later. :)

```python
if not (eqlims[0] < .... and ... and ...):
    continue
xdata = ...
....
ax.plot()
```

Reply:
You can combine some of these instead/as well:
```python
                if (eqlims[0] < magnitude < eqlims[1] and
                        eqlims[2] < distance < eqlims[3] and
                        min(ydata) < self.db_bin_edges[-1]): 
```
or
```python
                if not (eqlims[0] < magnitude < eqlims[1] and
                        eqlims[2] < distance < eqlims[3] and
                        min(ydata) < self.db_bin_edges[-1]): 
```